project-codeflare/instascale
# add: integration with helm for deployment

# Issue link
fixes #12

# What changes have been made
This PR integrates helm to manage the deployment process of instascale. This change is in line with the current deployment process of MCAD and aims to bring more consistency across the projects.

This pr also updates the documentation regarding deployment.

# Verification steps
-  Install the Helm Client on your local machine [here](https://docs.helm.sh/using_helm/#installing-helm). 

When deploying instascale to an OSD cluster:
```
cd deployment 
helm install instascale instascale
```

Then check to ensure each of the following have been deployed correctly and are operational:
instascale-configmap
instascale-sa
instascale-crb
instascale-cr
instascale

## Checks
- Testing Strategy
